src/openforms/forms/models/form_variable.py

```diff
@@ -98,6 +105,15 @@ def create_for_formstep(self, form_step: "FormStep") -> List["FormVariable"]:
                 )
             )
 
+            if fix_prefill_config:
+                for v in form_variables:
+                    if v.prefill_plugin != "" and v.prefill_attribute == "":
+                        logger.warning("Invalid prefill configuration: plugin {} specified without attribute".format(v.prefill_plugin))
```

Comment:
for future - do not manually interpolate when logging, leave that to the logging library:

```suggestion
                        logger.warning(
                            "Invalid prefill configuration: plugin %s specified without attribute",
                            v.prefill_plugin,
                        )
```